src/components/core/PivotTable.tsx

```diff
@@ -149,6 +152,9 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
     private lastScrollTop: number = 0;
     private lastScrollLeft: number = 0;
 
+    private watchingIntervalId: number = null;
+    private watchingTimeoutId: number = null;
```

Comment:
Can we remove the default values? The variables have type of number, but have ```null``` as default feels awkward for me.

Reply:
Done